server/src/main/java/com/flipkart/varadhi/web/AuthorizationHandlerBuilder.java

```diff
@@ -23,18 +25,28 @@
 public class AuthorizationHandlerBuilder {
 
     private final AuthorizationProvider provider;
+    private final Timer timer;
 
-    public AuthorizationHandlerBuilder(AuthorizationProvider provider) {
+    private final MeterRegistry meterRegistry;
+
+    public AuthorizationHandlerBuilder(AuthorizationProvider provider, MeterRegistry meterRegistry) {
         this.provider = Objects.requireNonNull(provider, "Authorization Provider is null");
+        this.meterRegistry = Objects.requireNonNull(meterRegistry, "Meter registry is null");
+        this.timer = Timer.builder("authorization.timer")
```

Comment:
lets create a small task on conventions on the metrics names. that is not decided yet. this name is fine for now as I dont have a strong opinion yet.

Reply:
Sure. will add this in the backlog